test_framework/scripts/longhorn-setup.sh

```diff
@@ -83,7 +83,7 @@ else
 fi
 
 
-if [[ ${LONGHORN_INFRA_TEST} ]] ; then
+if ${LONGHORN_INFRA_TEST} ; then
```

Comment:
Need a better way to ensure:
1. $LONGHORN_INFRA_TEST is true/True, `if` executes
2. Otherwise, `if` shouldn't

Now if the variable is an invalid value Or `True`, the script will error out.

Reply:
@yasker fixed